src/dmqmc_data.f90

```diff
@@ -9,10 +9,12 @@ module dmqmc_data
 ! The following indicies are used to access components of DMQMC numerators.
 enum, bind(c)
     enumerator :: energy_ind = 1
+    enumerator :: energy_imag_ind
     enumerator :: energy_squared_ind
     enumerator :: correlation_fn_ind
     enumerator :: staggered_mag_ind
     enumerator :: full_r2_ind
+    enumerator :: full_r2_imag_ind
```

Comment:
It's fine to leave the implementation of other estimators for complex calculations till later, but you should add some checks to make sure the user doesn't try to calculate something which doesn't have complex estimators implemented.

Reply:
All three operators for read_in systems are supported. Hence, the input checking for complex read_in DMQMC would be the same as that of real read_in DMQMC.